libs/execution_context/ebpf_program.c

```diff
@@ -1478,15 +1477,16 @@ _ebpf_helper_id_to_index_compare(const void* lhs, const void* rhs)
 
 /**
  * @brief Compute the hash of the program info and compare it with the hash stored in the program. If the hash does not
- * match then the program was verified against the wrong program info.
+ * match then the program was verified against the wrong program info. If the hash is not present then store the hash
+ * in the program to be used for future verification.
```

Comment:
What does "future verification" mean?  Another pass by the verifier?

Reply:
Improved the text for clarity.